.clang-format

```diff
@@ -0,0 +1,3 @@
+Language:        JavaScript
```

Comment:
@ofrobots do we auto-generate these now?  Can we delete them, or should they stay here?

Reply:
We do autogenerate these. There is no harm if you delete them, as we fall back to the default, but I would suggest keeping them. The benefit is that with these files present the editors know what formatting they should use as you edit.